src/os/bluestore/BlueStore.cc

```diff
@@ -17764,9 +17764,11 @@ void BlueStore::_log_alerts(osd_alert_list_t& alerts)
 void BlueStore::_collect_allocation_stats(uint64_t need, uint32_t alloc_size,
                                           const PExtentVector& extents)
 {
-  alloc_stats_count++;
-  alloc_stats_fragments += extents.size();
-  alloc_stats_size += need;
+  if (alloc_size != min_alloc_size) {
```

Comment:
I do not understand the exclusion that has been make here.
Even for smaller allocation we we still did make an allocation; and it has size.
Granted, the count of fragments will be 1.
How is counting that in obfuscating the allocation report?

Reply:
The primary purpose of these allocation stats/probes is to get some overview on how fragmented allocations get over time. Single unit allocations make no sense for this purpose and just spoil these stats. 
Hence removed.